UIKitDZ/UIKitDZ/CaloriesViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  CaloriesViewController.swift
+//  UIKitDZ
+//
+//  Created by Maxim on 02.10.2022.
+//
+
+import UIKit
+/// Контроллер колорийности
+class CaloriesViewController: UIViewController {
+
+    let backButton = UIButton(type: .roundedRect)
```

Comment:
```suggestion
   private let backButton = UIButton(type: .roundedRect)
```
private - пише всегда